docs/docs/plugins-themes-and-starters.md

```diff
@@ -36,3 +36,72 @@ Gatsby offers [official starters](/docs/starters/#official-starters) for a defau
 - [Starter library](/starters/)
 - [Creating a starter](/docs/creating-a-starter/)
 - [Converting a starter to a theme](/docs/themes/converting-a-starter/)
+
+## Comparing differences
+
+Once you have a conceptual understanding of what a plugin, theme, and starter is, you may ask yourself what each is suited for in relation to each other.
+
+The following tables put plugins, themes, and starters side-by-side, showing what each is more appropriate for.
+
+**Legend**
+
+| Icon | Feature Capability    |
+| ---- | --------------------- |
+| ●    | Fully configurable    |
+| ◐    | Somewhat configurable |
+| ○    | Not configurable      |
+
+### Differences in maintainability
+
+When it comes to maintaining a Gatsby site, plugins and themes offer a distinct advantage to starters by being distributed as packages, making pulling in upstream changes in multiple Gatsby sites as easy as a new install.
+
+| Maintenance        | Plugin | Theme | Starter |
+| ------------------ | ------ | ----- | ------- |
+| Versioning         | ●      | ●     | ●       |
+| Install as Package | ●      | ●     | ○       |
+
+#### Versioning
+
+Starters can still be versioned inside a repository so that you could track issues or bugs associated with specific developments in the starter but it's more difficult to make updates with new changes.
+
+#### Installing as a package
+
+Because starters are "ejected" each time they are used, it's difficult to sync changes across multiple sites derived from the same starter. This problem was one of the motivating factors in developing the newer concept of themes for Gatsby. You can read more about why themes were developed in the [What are Gatsby Themes guide](/docs/themes/what-are-gatsby-themes/#gatsby-starters).
+
+### Differences in configurability
+
+Plugins and themes can expose a varying level of an API surface (meaning a piece of the plugin or theme that a user of it can interact with), constraining what is possible and providing sensible defaults to make them easier to start picking up and using.
```

Comment:
```suggestion
Plugins and themes can expose varying levels of an API surface (meaning a piece of the plugin or theme that a user of it can interact with), constraining what is possible and providing sensible defaults to make them easier to start picking up and using.
```

Not sure if this is the right recommendation. Found this sentence a bit confusing.

Reply:
Yeah it doesn't read super great 🤔 I'll try and rework it, it's pretty long winded.